### DISCUSSION TITLE
AC's Performance Monitor

### DISCUSSION THREAD

**Discussion by viper-client** [seerge/g-helper#3536] [Q&A] [answered]
hey, appreciate seerge for making this amazing app although i guess something is missing or maybe i didnt notice. back in AC, there used to be a feature called "performance monitor" if im not wrong which had a customizable overlay. It was very handy for me since i could monitor temps, usages, frequencies and fan speed + showing the local time on top. Does G helper have this feature? if not, should i install AC again to use the overlay?

![Screenshot 2024-03-17 161414](https://github.com/user-attachments/assets/96444101-03a7-430a-a348-8bf4b25d24fb)

**Comment by seerge** [comment, marked as answer]
@viper-client hello, currently there is no overlay that would show those stats in the app.

However you can use bunch of existing alternatives. For example NVidia overlay from Nvidia Experience, AMD overlay from AMD Adrenaline (if you have AMD), [Intel Present Mon](https://game.intel.com/nl/intel-presentmon/) if you have Intel, or a popular overlay from 3rd party MSI Afterburner or Rivatuner apps.

**Reply by viper-client** [reply]
Thanks @seerge 
By the way, any chance you possibly consider adding it to G-helper in the near future as ghelper suppose to be an alternative for AC?

**Reply by seerge** [reply]
@viper-client may be, but actually it's not a trivial task due to windows own restrictions. As system simply doesn't allow to show info on top of everything else, unless app is signed with (expensive) certificate + does couple of other things :)

**Reply by viper-client** [reply]
@seerge appreciated, i may actually pay the price for it hahaha. Keep up the good work
